src/test/java/io/xstefank/wildfly/bot/utils/TestConstants.java

```diff
@@ -0,0 +1,10 @@
+package io.xstefank.wildfly.bot.utils;
+
+public class TestConstants {
+
+    public static final String TEST_REPO = "xstefank/wildfly";
+    public static final String PR_JSON = "valid-pr-template.json";
```

Comment:
```suggestion
    public static final String VALID_JSON_PR = "valid-pr-template.json";
```
or something like that. Name of the file itself is not as important as the used constant